Source/WebKit/WebProcess/InjectedBundle/API/Cocoa/WKWebProcessPlugInFrame.mm

```diff
@@ -117,6 +117,8 @@ - (JSValue *)jsRangeForRangeHandle:(WKWebProcessPlugInRangeHandle *)rangeHandle
 
 - (WKWebProcessPlugInBrowserContextController *)_browserContextController
 {
+    if (!_frame->page())
+        return nullptr;
```

Comment:
`return nil`?

Reply:
Does it matter? I'm not sure what our policy is for us cases.

Reply:
I think it's just convention to use nil for objective c objects. They don't have essential difference, according to objc.h:
```
#ifndef nil
# if __has_feature(cxx_nullptr)
#   define nil nullptr
# else
#   define nil __DARWIN_NULL
# endif
#endif
```